### DISCUSSION TITLE
Create an inventory of existing WZD Feeds

### DISCUSSION THREAD

**Discussion by DeraldDudley** [usdot-jpo-ode/jpo-wzdx#18] [General]
Publish where consumers can find existing WZD feeds

**Comment by antrim** [comment]
That seems essential!

With the [General Bikeshare Feed Specification](https://github.com/NABSA/gbfs) or GBFS, the inventory of feeds is in the same repo as the specification. This poses an issue because the [commit log](https://github.com/NABSA/gbfs/commits/master) mixes spec changes and feed location updates, so it's harder to isolate either type of change. When this is implemented, I'd recommend creating a separate but related repo that provides a directory of WZD feeds.

**Comment by chuehlien** [comment]
@DeraldDudley @antrim Thank you both for this feedback! I've started https://github.com/usdot-jpo-ode/jpo-wzdx/issues/49 to get feedback on a few upcoming features for consumers of realtime/historical WZDx data, one of which is an inventory of active WZDx feeds. Would love to get your feedback on it there and discuss more.

Julia Lien, USDOT ITS DataHub
